THIRD-PARTY

```diff
@@ -0,0 +1,71 @@
+ISC License
```

Comment:
Could you add which dependency this license is for?

Reply:
Done